assets/sass/v2/_footer.scss

```diff
@@ -1,12 +1,35 @@
 .siw-main-footer {
   .auth-footer {
     max-width: 100%;
-    width: max-content;
     display: flex;
     flex-direction: column;
     .link {
+      width: max-content;
```

Comment:
Just making this doesn't cause regressions, does a very long link label still wrap to the next line?

Reply:
I don't think it's going to wrap.. But it's a good point. We should truncate such long links/label instead of wrapping into the second line. Luckily we don't have such long links or labels right now.

Reply:
It's supposed to wrap. v1 wraps. The labels can be customized so a user could set it to something very long, and in this case we don't want to show a horizontal scroll bar. https://oktainc.atlassian.net/browse/OKTA-395123